evap/staff/views.py

```diff
@@ -1308,6 +1308,19 @@ def questionnaire_update_indices(request):
     return HttpResponse()
 
 
+@require_POST
+@manager_required
+def questionnaire_visibility(request):
+    questionnaire_id = request.POST.get("questionnaire_id")
+    visibility = int(request.POST.get("visibility"))
```

Comment:
These could in theory be `None` which would then lead to a 500.

Reply:
but then someone would have tempered with the post data or didn't send the required data in the request, right? so seems okay for me.

Reply:
Fair enough.